.travis.yml

```diff
@@ -31,4 +31,4 @@ script:
   - cat sql/updates/hotfixes/*.sql | mysql -utrinity -ptrinity hotfixes
   - mysql -uroot < sql/create/drop_mysql.sql
   - cd bin
-  - make -j 10
+  - make -j 8
```

Comment:
This change is to prevent travis running out of memory, and clang being killed as of boost graph templates, just in case someone wonders why I change travis build parallel number